ledger/controllers/publisher.js

```diff
@@ -640,16 +640,14 @@ module.exports.initialize = async (debug, runtime) => {
       unique: [ { rulesetId: 1 } ],
       others: [ { type: 1 }, { version: 1 }, { timestamp: 1 } ]
     },
-/* deprecated
     {
       category: runtime.database.get('publishers', debug),
```

Comment:
should this be `get('publishersX'...` ? we have two collections in use as far as I can tell - `publishersV2` and `publishersX`. additionally there is one legacy collection `publishers`.

the code here looks like it's creating an index `publishersX` on the legacy collection `publishers`. is that right? seems confusing to reuse the collection name as an index name on a different collection.

Reply:
sorry, my fault: `publishers` became `publishersX` and the edit below should have just just included the rename rather that commenting it out. i botched that.